functionalTest/test/specs/example.e2e.js

```diff
@@ -1,13 +1,13 @@
 import dashboard from "../pageobjects/dashboardPage";
 
 describe("Covid-19 Dashboard", () => {
-  it("should have correct title", () => {
+  xit("should have correct title", () => {
```

Comment:
Looks like all of these tests have been disabled. If they're functional, have them enabled in the PR. If not, explain why in a dated comment so people know how long a test has been dead for

Reply:
Also, the PR should always be ready to be merged to master. There's a bunch of disabled test cases which I'm guessing were disabled so's the test run of your tests didn't take so long. That's fine during dev, but the disabling shouldn't be committed/pushed

Reply:
I'll enable them all and push again.